packages/back-end/types/stats.d.ts

```diff
@@ -60,15 +68,24 @@ export type ExperimentMetricAnalysis = {
   }[];
 }[];
 
-export type BanditResponse = {
-  banditUpdateMessage?: string;
+export type SingleVariationResult = {
+  users?: number;
+  cr?: number;
+  ci?: [number, number];
+};
+
+export type BanditResult = {
+  singleVariationResults?: SingleVariationResult[];
   banditWeights?: number[];
+  bestArmProbabilities?: number[];
   additionalReward: number;
+  seed: number;
+  banditUpdateMessage?: string;
```

Comment:
`banditError?: string;` for anything that breaks the bandit

Reply:
added `banditError?: string;`